mathics/eval/arithmetic.py

```diff
@@ -319,48 +321,27 @@ def eval_complex_sign(n: BaseElement) -> Optional[BaseElement]:
     sign = eval_RealSign(expr)
```

Comment:
I now see that SymPy has a "Sign" function that works over reals and complexes. 

I missed this fact before.  We should be using that.

Reply:
`sympy.Sign` works like WL `Sign` and, even if it is true to force `sympy.Sign` as WL `RealSign`, it would mean doing an extra work in conversion, by checking that the expression is a Real expression. This can be done by hacking `to_sympy` to handle `RealSign` in a special way, but that work is more or less equivalent to performing the actual computation in pure Python, with the overhead of the back-and-forth conversion.

Reply:
BTW, `Sign[3 - 4 I]` can not be evaluated by using only sympy. The problems of using `sympy.sign` are shown in the tests by replacing the implementation of `mathics.eval.arithmetic.eval_Sign` by

```
def eval_Sign(expr: BaseElement) -> Optional[BaseElement]:
    """
    if expr is a number, return its sign.
    """
    sympy_expr = expr.to_sympy()
    if sympy_expr is not None:
        print("   sympy_expr", sympy_expr)
        result = from_sympy(sympy.sign(sympy_expr))
        if result is not None:
            return result
    return None

```


In a similar way, if we have expressions with products of  negative rational powers, the work of convert back the product to Mathics is more or less the same than evaluating the product without sympy.

Reply:
Here is what I see when I evaluate the Python/Sympy equivalent of `Sign[3 - 4I]`: 

```
>>> from sympy import *
>>> sign(3 - 4j)
0.6 - 0.8*I
>>> 
````

It looks like Complex.to_sympy() is not doing the right thing.